recipes/ampliconsuite/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# pull source code for AA and move scripts into a library
+TARGET="AmpliconArchitect"
+TARGET_VERSION="1.3.r5"
+wget https://github.com/AmpliconSuite/${TARGET}/archive/refs/tags/v${TARGET_VERSION}.zip
```

Comment:
IMO this should be realized as additional sources in meta.yaml, and adding `__init__.py files as patches.

Reply:
If amplicon architect and classifier are also of interest stand alone a separate recipe might also be a good option.

Reply:
Hi Matthias - thanks so much for reviewing this PR. Regarding your second comment - we would prefer if most users not attempt to use the AmpliconArchitect and AmpliconClassifier modules in a standalone fashion. The functionality from both can be accessed from `AmpliconSuite-pipeline.py`. That is the single entrypoint we would like to support going forward, as it captures our best practices for using the encompassed modules.

I will update the recipe with your excellent suggestions and get back to you shortly.

Thanks again,
Jens